### PR TITLE
Chekbox styles

### PR DESCRIPTION
**Pull request by aguspina**
Merge after #698

### CODE REVIEW

**Comment by aguspina** [comment]
screen shot 2016-11-21 at 11 34 32 am

**Comment by aguspina** [comment]
screen shot 2016-11-21 at 12 07 51 pm

File: app/assets/stylesheets/application/modules/_checkboxes.scss [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        color: white;
+        background-color: $brand-complementary;
+        border-radius: 1em;
+      }
+      &:not(:checked) ~ label:before {
+        border-radius: 1em;
+      }
+    }
+    input[type="checkbox"] {
+      &:checked ~ label:before {
```
Comment (flbulgarelli) [inline app/assets/stylesheets/application/modules/_checkboxes.scss]: Have you added those styles to `mumuki-styles`, right?

If that is true, please don't repeat the style logic here, and use the `mumuki-styles` bower component instead. Use [rails-assets](https://rails-assets.org/), in the same way as we used it to include the `devawesome` font.

**Comment by flbulgarelli** [comment]
@aguspina I think those radios and checkboxes are toooo big. 

Also, radios should have a dot instead of a check. 

Finally, I am wondering whether we could have pink circumferences instead of filled circles, and pink rectangles only for checked radios. 

Please check this:

*  http://flatlogic.github.io/awesome-bootstrap-checkbox/demo/  
* https://github.com/flatlogic/awesome-bootstrap-checkbox

I think we could even use that css bower component.

**Comment by aguspina** [comment]
screen shot 2016-11-24 at 11 26 55 am
screen shot 2016-11-24 at 11 28 53 am

**Comment by flbulgarelli** [comment]
I like those styles much more. 

However, please notice that radios are not aligned properly

**Comment by flbulgarelli** [comment]
PS: Please remove bower and use rails-assets instead

File: app/assets/stylesheets/application/modules/_checkboxes.scss [hunk 2]
```diff
@@ -0,0 +1,18 @@
+.complementary label::after {
+  background-color: $brand-complementary !important;
+  color: white !important;
+}
+
+.complementary-checkbox label::before {
+  bottom: 7px !important;
+}
```
Comment (flbulgarelli) [inline app/assets/stylesheets/application/modules/_checkboxes.scss]: Are those `!important` really necessary? They scary me

Comment (flbulgarelli) [inline app/assets/stylesheets/application/modules/_checkboxes.scss]: Perhaps it is just matter of reordering stylesheets

File: app/assets/stylesheets/application/modules/_checkboxes.scss [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+.complementary-checkbox label::after {
+  top: 6px !important;
+  width: 17px !important;
+  heigth: 17px !important;
+  line-height: 14px !important;
```
Comment (flbulgarelli) [inline app/assets/stylesheets/application/modules/_checkboxes.scss]: This css attribute is wrong

**Comment by aguspina** [comment]
@flbulgarelli please check now. I will move all styles to mumuki-styles in other PR

**Comment by flbulgarelli** [comment]
Ok. I still the bowerc file is not necessary. I am going to remove that file, and squash everything in order to maintain git repository lighter

**Pull request merged**